src/status_im/contexts/wallet/common/utils.cljs

```diff
@@ -478,3 +478,13 @@
    :toAsset     to-asset
    :fromAmount  amount-out
    :type        multi-transaction-type})
+
+(defn estimated-time-format
+  "Formats the estimated time for a transaction"
+  [estimated-time]
+  (condp = estimated-time
+    (:unknown constants/wallet-transaction-estimation)                 "> 5"
+    (:less-than-one-minute constants/wallet-transaction-estimation)    "< 1"
```

Comment:
maybe without the spaces here, as `> 5 min` looks a bit weird. Usually the "smaller than" is shown right next to the number like `>5 min`